src/js/themes/base.js

```diff
@@ -378,8 +378,29 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         radius: `${baseSpacing * 0.75}px`,
       },
       // color: { dark: undefined, light: undefined }
+      simple: {
```

Comment:
`simple`/`primary` & default buttons should include `disabled`, `active`, `hover` and `extend`.
Anything that is styled on the default button should have a style option on the simple/primary objects.
padding should also be an option for each type of button.